### PR TITLE
Layout improvements for desktop, episode 002 title

### PR DESCRIPTION
**Pull request by samuelfine**
Mostly just restricted the width to 960px and centered everything so it looks nice on wider screens while retaining fluidity at phone/tablet sizes.

Also gave the episode a title. ¯_(ツ)_/¯

### CODE REVIEW

File: source/layouts/layout.erb [hunk 1]
```diff
@@ -16,8 +16,10 @@
       <%= image_tag "logo-300.png", alt: "Logo" %>
       <h1><%= link_to site_name, "/" %></h1>
     </header>
-
-    <%= yield %>
+    
+    <main>
```
Comment (pbyrne) [inline source/layouts/layout.erb:20]: TIL that there's a `main` element in HTML5 now.

File: source/episodes/002.markdown [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 date: 2015-07-01
 excerpt: In this inaugural episode (pay no attention to the episode number behind the curtain), Pat and Sam discuss the first episode of Mr Robot. Critics are raving, but what think our intrepid hosts? Listen to find out!
-title: Mr Robot S01E01
+title: Unnatural Pause
```
Comment (pbyrne) [inline source/episodes/002.markdown:4]: Maybe I'm just too boring (or literal), but `Mr Robot S01E01` seemed _just perfect_ to me.

I'm torn now between taking this as-is or merging the two: `Unnatural Pause - Mr Robot S01E01`, in the interest of making the title both funny/memorable _and_ useful when looking back on older content. Though, perhaps, the better way of doing that is a show index (a la, all the Mr Robot episodes, all the Game of Thrones episodes, and such), which I think makes sense when we have many more episodes than what we have now.

File: source/stylesheets/_base.css.scss [hunk 3]
```diff
@@ -11,6 +11,12 @@ header {
   margin: 0px auto;
   max-width: 960px;
   text-align: center;
+  
+  img {
+    width: 25%;
```
Comment (pbyrne) [inline source/stylesheets/_base.css.scss:16]: I kinda dug how huge the image was on mobile, being full-width there. Was totally nutty on non-mobile, tho.

Comment (samuelfine) [inline source/stylesheets/_base.css.scss:16]: Feel free to tweak the 25% width value! It did look a little small on mobile.

Comment (pbyrne) [inline source/stylesheets/_base.css.scss:16]: Done: https://github.com/watershednetwork/somefine.tv/commit/8beb315133fd62d47414ebe455a93f702b2e542b.

**Pull request merged**